src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
```

Comment:
I think this sort of thing is ultimately subjective.  Unless some official Scala style guide has some very specific things to say about what should be on one line, can we just leave it up to the author to decide little things like this?  I kind of agree with @mcovarr that at some line length, it gets unwieldy, but my threshold may be different than yours.  I personally have no objection to how it is now, it's perfectly readable to me.